### PR TITLE
AddReverseBtnToDashboard

### LINKED ISSUE #101: Insert reverse topics order to MoodleBooster Dashboard
## What?

Insert the button of "reverse topics order" to MoodleBoosters Dashboard.

## Why?

To be more orginazied and, because the dashboard is currently empty while it's whole purpose is to include features like this.

Screen Shot 2021-06-21 at 11 47 43

### PR DESCRIPTION
**Pull request by guys1915**
detailed in #101

### CODE REVIEW

File: content_scripts/courseReverseOrder.js [hunk 1]
```diff
@@ -30,10 +30,10 @@ const addButtonStyle = () => {
 }
 
 const addCourseReverse = () => {
-    let courseContent = document.getElementsByClassName("course-content");
```
Comment (tamiror6) [inline content_scripts/courseReverseOrder.js:33]: nice!!

**Comment by idan2468** [comment]
see my comment on issue #101 before accepting the PR @norbit8 @guys1915

File: content_scripts/courseReverseOrder.js (hunk 1 above)
Comment (idan2468) [reply content_scripts/courseReverseOrder.js:33]: See comments on issue #101 
I think we want to revert it...
@tamiror6

**Comment by idan2468** [comment]
need to fix bug #97 before merging this PR

**Review by idan2468 — CHANGES_REQUESTED** [review]
need to fix bug #97 before merging

**Comment by guys1915** [comment]
@idan2468 I think we should merge this PR, and have the necessary changes in a different issue. WDYT?

**Comment by idan2468** [comment]
Fine by me

**Review by idan2468 — APPROVED** [review]

**Pull request merged**